ptx/S_12_3_ParamLineIntegrals.ptx

```diff
@@ -2,427 +2,264 @@
     </p>
-
-    <p>
-      This final Riemann sum is <m>\int_a^b \vF(\vr(t))\cdot \vr'(t)\, dt</m>,
-      allowing us to reduce the evaluation of a line integral of a
-      vector-valued function along an oriented curve to an ordinary integral
-      of a function of one variable,
-      since after evaluating the dot product,
-      <m>\vF(\vr(t))\cdot \vr'(t)</m> is (scalar) function of <m>t</m>.
-      We restate
-      this result below for easy reference.
-    </p>
-
-    <p>
-      Let <m>C</m> be a smooth, oriented curve traced out by the
-      vector-valued function <m>\vr(t)</m> for <m>a\leq t\leq b</m> and let
-      <m>\vF</m> be a continuous vector field defined near <m>C</m>.
-      Then
+    
+    <theorem xml:id="LineInt_Param_Thm">
+      <p>
+      Let <m>C</m> be a smooth, oriented curve traced out by the vector-valued function <m>\vr(t)</m> for <m>a\leq t\leq b</m> and let <m>\vF</m> be a continuous vector field defined near <m>C</m>. Then
       <me>
         \int_C \vF\cdot d\vr = \int_a^b \vF(\vr(t))\cdot \vr'(t)\, dt
       </me>.
-    </p>
-
+      </p>
+    </theorem>
     <example>
       <statement>
         <p>
-          Let <m>\vF(x,y) = x\vi + y^2\vj</m> and let <m>C</m> be the quarter of the circle of
-          radius <m>3</m> from <m>(0,3)</m> to <m>(3,0)</m>.
-          This vector field and curve are
-          shown in <xref ref="F_12_3_field-quarter-circle">Figure</xref>.
-          By properties of line integrals,
-          we know that <m>\int_C \vF\cdot d\vr = -\int_{-C}\vF\cdot d\vr</m>, and
-          we will use this property since <m>-C</m> is the usual clockwise
-          orientation of a circle, meaning we can parameterize <m>-C</m> by
-          <m>\vr(t)
-          = \langle 3\cos(t),3\sin(t)\rangle</m> for <m>0\leq t\leq \pi/2</m>.
+          Let <m>\vF(x,y) = x\vi + y^2\vj</m> and let <m>C</m> be the quarter of the circle of radius <m>3</m> from <m>(0,3)</m> to <m>(3,0)</m>. This vector field and curve are shown in <xref ref="F_12_3_field-quarter-circle">Figure</xref>. By properties of line integrals, we know that <m>\int_C \vF\cdot d\vr = -\int_{-C}\vF\cdot d\vr</m>, and we will use this property since <m>-C</m> is the usual clockwise orientation of a circle, meaning we can parameterize <m>-C</m> by <m>\vr(t) = \langle 3\cos(t),3\sin(t)\rangle</m> for <m>0\leq t\leq \pi/2</m>.
         </p>
 
         <figure xml:id="F_12_3_field-quarter-circle">
-          <caption>The vector field <m>\vF = x\vi + y^2\vj</m> and an oriented
-          curve <m>C</m></caption>
-          <image width="40%" source="fig_12_3_field_quarter_circle.pdf" />
+          <caption>The vector field <m>\vF = x\vi + y^2\vj</m> and an oriented curve <m>C</m> </caption>
+          <image source="images/fig_12_3_vf_quarter_circle"/>
```

Comment:
Generally should have `@width` attribute on `image` inside `figure`, as otherwise has a tendency to blow up to 100% of line width at the most inopportune moments.